buildSrc/src/main/java/org/opensearch/gradle/precommit/ValidateJsonAgainstSchemaTask.java

```diff
@@ -113,7 +114,7 @@ public void validate(InputChanges inputChanges) throws IOException {
                 File file = fileChange.getFile();
                 if (file.isDirectory() == false) {
                     // validate all files and hold on to errors for a complete report if there are failures
-                    getLogger().debug("Validating JSON [{}]", file.getName());
+                    getLogger().debug("Validating JSON: " + file.getName());
```

Comment:
Curious why this change? The previous version was more "lazy" and is not doing anything if the debug logging is not enabled.

Reply:
Oops, it is different because I had swapped to COUT for for reading the info and missed swapping it. I will fix that now.